src/daemon/config.js

```diff
@@ -47,7 +47,7 @@ function applyDefaults (ipfsd) {
   config.Swarm = config.Swarm || {}
   config.Swarm.DisableNatPortMap = false
   config.Swarm.ConnMgr = config.Swarm.ConnMgr || {}
-  config.Swarm.ConnMgr.GracePeriod = '1m'
+  config.Swarm.ConnMgr.GracePeriod = '3m'
   config.Swarm.ConnMgr.LowWater = 20
   config.Swarm.ConnMgr.HighWater = 40
```

Comment:
@lidel Are daemon values configurable in the app? If not, is there a reason other than dev bandwidth?

Reply:
@SgtPooki yes. You can modify the whole config file of the daemon.

dev bandwidth?

Reply:
Yes, we simply had no bandwidth to work on this + waiting for resource manager to ship in go-ipfs 0.13, as noted in https://github.com/ipfs/ipfs-desktop/pull/2096#pullrequestreview-936752564

If you want to pick this up, bump ipfs-desktop to 0.13-rc1 as part of this PR.

Reply:
- Let's keep this PR to be about one thing: `Swarm.ConnMgr.GracePeriod`
- I don't understand how LAN dials could kill routers, afaik it is high churn of WAN connections that is  the usual problem, but  if you have additional information please comment in https://github.com/ipfs/go-ipfs/issues/3320 
- Pubsub is tracked in https://github.com/ipfs/go-ipfs/issues/6621 and  https://github.com/ipfs/go-ipfs/issues/8591 – no updates.
- Hole punching will be enabled  in https://github.com/ipfs/ipfs-desktop/issues/1946#issuecomment-1131511214, no need to do anything